client/data/emails/use-remove-email-forward-mutation.ts

```diff
@@ -1,24 +1,35 @@
+import { useTranslate } from 'i18n-calypso';
+import { createElement } from 'react';
 import { useMutation, useQueryClient } from 'react-query';
-import { useSelector } from 'react-redux';
+import { useDispatch, useSelector } from 'react-redux';
 import { getCacheKey as getEmailDomainsQueryKey } from 'calypso/data/domains/use-get-domains-query';
+import { getEmailAddress } from 'calypso/lib/emails';
 import wp from 'calypso/lib/wp';
+import { errorNotice, successNotice } from 'calypso/state/notices/actions';
 import { getSelectedSiteId } from 'calypso/state/ui/selectors';
 import { getCacheKey as getEmailAccountsQueryKey } from './use-get-email-accounts-query';
-import type { UseMutationOptions, UseMutationResult } from 'react-query';
+import type { Mailbox } from './types';
+import type { UseMutationOptions } from 'react-query';
+
+type Context = {
+	[ key: string ]: any;
+};
+
+const MUTATION_KEY = 'removeEmailForward';
 
 /**
  * Deletes an email forward
```

Comment:
Not mandatory but just to be more consistent with the new mutation hook:

```suggestion
 * Deletes an email forward, including relevant optimistic data mutations.
```